integration-tests/infinispan-client/src/main/java/io/quarkus/it/infinispan/client/TestServlet.java

```diff
@@ -254,8 +248,8 @@ public Response createItem(String value, @PathParam("id") String id) {
     @Path("magazinequery/{id}")
     @GET
     public String magazineQuery(@PathParam("id") String name) {
-        QueryFactory queryFactory = Search.getQueryFactory(magazineCache);
```

Comment:
@gsmet this code will break now but is a mandatory change. As you see, the query code remain the same, but executed directly from the remote cache api instead of the QueryFactory indirection